runtime/src/lib.rs

```diff
@@ -4,10 +4,16 @@ mod logger;
 mod next;
 mod provisioner_factory;
 
+pub use async_trait::async_trait;
 pub use legacy::{start, Legacy};
 pub use logger::Logger;
 #[cfg(feature = "next")]
 pub use next::{AxumWasm, NextArgs};
 pub use provisioner_factory::ProvisionerFactory;
 pub use shuttle_common::storage_manager::StorageManager;
-pub use shuttle_service::{main, Error, Service};
+pub use shuttle_service::{main, Error, ResourceBuilder, Service};
+
+pub type CustomError = anyhow::Error;
+pub use anyhow::Context;
+pub use tracing;
+pub use tracing_subscriber;
```

Comment:
Do we remove these from service now?

Reply:
They aren't used for anything else, so it makes sense to me :+1: